client/IndexComponent.js

```diff
@@ -2,6 +2,8 @@ import React, {Component} from 'react';
 import {Route, BrowserRouter, Switch} from 'react-router-dom';
 
 import Login from '../imports/components/LoginComponent';
+import Bucket from '../imports/ui/buckets/BucketView';
```

Comment:
Let's try to be consistent - here we're importing "Bucket" and below "ProfileView", where the original files both have "View" in the title.

Reply:
I hear what you're saying here, but keep in mind these imports are used for testing components until routing is implemented.